java/junit5/src/main/java/com/saucelabs/saucebindings/junit5/SauceBindingsExtension.java

```diff
@@ -106,7 +106,7 @@ public void testFailed(ExtensionContext context, Throwable cause) {
 
   // TODO: Implement this in SauceSession directly
   private boolean isExtensionDisabled() {
-    String value = System.getenv("SAUCE_DISABLED");
+    String value = System.getProperty("SAUCE_DISABLED");
```

Comment:
Our implementation of `getenv()` method checks both property and environment variable

Reply:
oh, maybe that was something special in SauceBindings not this library